### PR TITLE
Game Play Information

### PR DESCRIPTION
**Pull request by dehli**
# Problem
When you go to a game's page, you don't get enough information sent back (current arena state if the game isn't running, game name, max players, current players, etc.).

# Solution
With this change, I broadcast game info, through sockets, whenever somebody joins a game. I'm not sure that we really need to broadcast it to everyone (could probably get away with just broadcasting to the channel that joined). We'll need to think about this. I still need to send a `stats-update` so that the client knows which bots are in the game.

### CODE REVIEW

File: src/wombats/sockets/game.clj [hunk 1]
```diff
@@ -157,11 +188,23 @@
...
+          arena (get-in game-state [:frame :frame/arena])]
+
       (swap! game-rooms
              assoc-in
              [game-id :players chan-id]
-             (assoc socket-user :color (:player/color player))))))
+             (assoc socket-user :color (:player/color player)))
+
+      ;; Sends the initial frame to the frontend (TODO: do we really need these separated though?)
+      (broadcast-arena game-id arena)
```
Comment (dehli) [inline src/wombats/sockets/game.clj]: We probably will want to skip the arena/player broadcasts if the game is in progress. Would like to talk through this tomorrow.

Comment (oconn) [inline src/wombats/sockets/game.clj]: Yeah I could see it working both ways, you'll get a frame update within a second or two anyway so we could ignore it. On the other hand, you'll get an immediate paint if we send it right away.

**Comment by dehli** [comment]
I made some changes to the PR. Now only the channel that is joining a game gets the info broadcasted to it. Additionally, I now broadcast user stats as well. @emilyseibert @oconn

**Comment by emilyseibert** [comment]
No it should start at 100, full health.

**Comment by dehli** [comment]
Totally agree with you about file size. When the game ends, it looks like the existing stats functions make the hp go to 0. Will investigate 🕵️

**Pull request merged**